terraform/modules/gke/main.tf

```diff
@@ -72,11 +74,51 @@ module "gke_cluster" {
   }
 }
 
+#resource "google_container_node_pool" "general" {
+#  name       = "general"
+#  cluster    = google_container_cluster.primary.id
+#  node_count = 1
+#
+#  management {
+#    auto_repair  = true
+#    auto_upgrade = true
+#  }
+#
+#  node_config {
+#    preemptible  = false
+#    machine_type = "e2-small"
+#
+#    labels = {
+#      role = "general"
+#    }
+#
+#    service_account = google_service_account.kubernetes.email
+#    oauth_scopes = [
+#      "https://www.googleapis.com/auth/cloud-platform"
+#    ]
+#  }
+#}
+
```

Comment:
why comment this out?

Reply:
The recommendation is to create a node-pool separately so changing it does not re-create the existing cluster. I will add a TODO comment here